apps/pie-docs/src/_includes/home-page.njk

```diff
@@ -0,0 +1,30 @@
+{% set heroSrc = ['/assets/img/heroes/', heroImageSrc] | join | url %}
+
+<div class="c-homePage" data-test-id="site_content">
+    <div class="c-homePage-header">
+        <div class="c-content-container">
+            <div class="c-homePage-heading">
+                <div class="c-homePage-title">
+                    <h1 class="alpha--standout">
+                        {{ title }}
+                    </h1>
+                    <p class="c-contentPage-header-description">
+                        {{ description }}
+                    </p>
+                    <a class='o-btn c-homePage-button' href={{ heroCtaHref | url }}>{{ heroCtaTitle }}</a>
+                </div>
+                <div class="u-showAboveMid c-homePage-image">
+                    <img src={{ heroSrc }} alt="" />
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="razor"></div>
+    <div class="c-homePage-content">
+        <div class="c-contentPage-content">
```

Comment:
The class naming here is a bit confusing – using `c-contentPage-content` when this isn't a `contentPage`.

The intention would be that this class would be a child of `c-contentPage` but this is a child of `c-homePage`, so may need a bit of renaming here to make this a bit clearer. I'd personally update them so that they are less page dependent (maybe if this is styling content across pages, it could be named `c-content` and then `c-content-container` makes sense being used here too.